### ISSUE TITLE
xanthine and hypoxanthine

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7541]
**Preferred term label:**
Increased serum and urinary hypoxanthine
Increased serum and urinary xanthine 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For annotation of 603592 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
HPO has Hyperxanthinemia HP:0010933
Xanthinuria HP:0010934
Increased urinary hypoxanthine HP:0011814

-- added the missing term Increased circulating hypoxanthine concentration
